apis/python/tests/ht/test_ht_dataframe.py

```diff
@@ -241,7 +240,6 @@ def dataframe_domain(
     NB:
```

Comment:
Given that we now use Arrow timestamps as the "native" domain type, I suggest converting this class to do the same (i.e., `self.domain` should be a tuple of pa.TimestampScalar, not np.datetime64).  That would remove the need to do all the conversions like line 254 (`zero = ...`)